platforms/Windows/bundle/theme.xml

```diff
@@ -9,7 +9,7 @@
     <Font Id="2" Height="-22" Weight="500" Foreground="graytext">Segoe UI</Font>
     <Font Id="3" Height="-12" Weight="500" Foreground="windowtext" Background="window">Segoe UI</Font>
 
-    <Window Width="614" Height="384" HexStyle="100a0000" FontId="0" Caption="#(loc.Caption)">
+    <Window Width="614" Height="456" HexStyle="100a0000" FontId="0" Caption="#(loc.Caption)">
         <ImageControl X="0" Y="0" Width="165" Height="384" ImageFile="swift_side.png"/>
```

Comment:
New side banner from ##298 would need this height to be adjusted as well
```suggestion
        <ImageControl X="0" Y="0" Width="165" Height="456" ImageFile="swift_side.png"/>
```

Reply:
Tested it , works great!